src/VsixV3/Microsoft.VisualStudio.Templates.CSharp.NetStandard.vsmanproj

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\build\Targets\VsixV3.Settings.props" />
+  <PropertyGroup>
+    <FinalizeManifest>true</FinalizeManifest>
+    <FinalizeSkipLayout>true</FinalizeSkipLayout>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <OutputPath>$(OutDir)\VsixV3</OutputPath>
+    <IsPackage>true</IsPackage>
+    <UseVisualStudioVersion>true</UseVisualStudioVersion>
+    <FinalizeValidate>false</FinalizeValidate>
+    <ValidateManifest>false</ValidateManifest>
+  </PropertyGroup>
+
+  <Import Project="$(NuGet_Packages)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
+  <Import Project="$(NuGet_Packages)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.targets" />
+
+  <Target Name="BeforeBuild">
+    <MSBuild Projects="Microsoft.VisualStudio.Templates.CSharp.NetStandard.swixproj" Targets="Build" />
+    <MSBuild Projects="Microsoft.NET.Sdk.swixproj" Targets="Build" />
+  </Target>
+
+  <ItemGroup>
+    <MergeManifest Include="$(OutputPath)\Microsoft.VisualStudio.Templates.CSharp.NetStandard.json" />
+    <MergeManifest Include="$(OutputPath)\Microsoft.NET.Sdk.json" />
```

Comment:
This shouldn't include the NET SDK - otherwise the same component is being included in two vsmanprojs. Ideally we should split out the SDK into a separate vsmanproj later when we move to the model for pulling in unzipped sdks.